index.html

```diff
@@ -146,48 +185,55 @@ <h1 class="about-me-title">About me</h1>
     </p>
     <button class="about-me-button">Get My Resume</button>
     <div class="about-me-line"></div>
-    <ul class="about-me-lists">
-      <li class="about-me-list-header">
```

Comment:
Please, also adjust the about section to match that of your chosen design template. Kindly check the comment above for more reason you should consider sticking closely to a given design.

Your design: 
![image](https://user-images.githubusercontent.com/20554583/187029541-328ea441-704d-44ae-8e39-5fb351654126.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/187029524-d7f71ce8-b344-4707-8eef-743518d5eb55.png)